src/ruby.ts

```diff
@@ -35,6 +39,12 @@ export class Ruby {
   ) {
     this.context = context;
     this.workingFolder = workingFolder;
+    if (os.platform().indexOf('win') == 0) {
+      this.shell = "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
```

Comment:
Why do we need to set the shell manually? Does Windows not set this in any way?

Reply:
I use powershell here, because otherwise node would use cmd.exe which makes problems with the string, can't escape " (double quotes) in cmd.exe. So wither the cmd string breaks, or the ruby string.